docker/Dockerfile

```diff
@@ -68,7 +71,7 @@ RUN \
     cd ${dir}/${repo_scim}-${tag} && \
     source ./${pack_scim} --with-python27 --with-version $KEYSTONE_SCIM_VERSION --with-release 0 && \
     find . -name "*.rpm" -exec rpm -Uvh {} \; && \
-    # Install Keystone-SPASSWORD
+     # Install Keystone-SPASSWORD
```

Comment:
Miss indented?

Reply:
fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/136/commits/25a077d47c9688b78c434359e78411fb11a6ba39